### PR TITLE
Add preconfigured UICR hex file for validation and makefile commands …

### PR DESCRIPTION
**Pull request by kylzrnr**
I've created a hex file that stores the address and data information for the bit that corresponds to validation mode. I've also added some makefile commands to merge this file with the release firmware hex file, flash it to a board, and zip it.

I tested that it does in fact set the correct bit by implementing it locally on my validation branch, but I wanted to push it here so that the code is contained within this particular issue.

### CODE REVIEW

**Comment by kylzrnr** [comment]
I accidentally pushed my change to the SDK_ROOT address. I pushed a new commit to correct it.

File: Makefile [hunk 1]
```diff
@@ -442,6 +443,23 @@ hex_factory: firmware_factory settings_factory
 flash_factory: erase hex_factory
 	nrfjprog -f nrf52 --program $(OUTPUT_DIRECTORY)/full_firmware_factory.hex --sectorerase --verify --reset
 
+#
+# Validation commands
+#
+
+.PHONY: hex_validation
+hex_validation: firmware_release settings_release
+	mergehex -m $(OUTPUT_DIRECTORY)/firmware.hex $(OUTPUT_DIRECTORY)/firmware_settings.hex $(SOFTDEVICE_HEX_PATHNAME) $(BOOTLOADER_HEX_PATHNAME) -o $(OUTPUT_DIRECTORY)/full_firmware.hex
```
Comment (obasille) [inline Makefile:452]: Can you make `hex_validation` call `hex_release` to build `full_firmware.hex`?

**Review by obasille — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (kylzrnr) [reply Makefile:452]: Sure, I'll do that right now!

Comment (kylzrnr) [reply Makefile]: Pushed!

**Review by obasille — APPROVED** [review]

**Review by obasille — CHANGES_REQUESTED** [review]
Hi Kyle!

I've tried this command:
make flash_validation

Got this message
WARNING: A UICR write operation has been requested but UICR has not been
WARNING: erased. Please verify that the result is correct.

And indeed the firmware ended up in normal mode

**Review by obasille — CHANGES_REQUESTED** [review]
Can you describe how you build the UICR hex file in the Makefile (like what you send me by chat but shorter, with a link to hex format description), where it's being used and rename the hex file to something more explicit like "UICR_ValidationModeEnabled" for example.

**Comment by kylzrnr** [comment]
Hmmm that is odd. It worked for me before I committed. I will double check what could be going on and get back to you. I will add the comments you requested as well.

**Comment by obasille** [comment]
Got this error on a clean clone (you'll probably get the same by deleting the _build folder)

nrfutil settings generate --family NRF52810 --application-version 0x1 --bootloader-version 0x1 --bl-settings-version 1 --application _build/firmware.hex _build/firmware_settings.hex
process_begin: CreateProcess(NULL, nrfutil settings generate --family NRF52810 --application-version 0x1 --bootloader-version 0x1 --bl-settings-version 1 --application _build/firmware.hex _build/firmware_settings.hex, ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [Makefile:347: settings] Error 2

**Comment by kylzrnr** [comment]
I eliminated the warning message about requesting a UICR write without erasing by changing the `--sectorerase` flag in the flash_validation call to nrfjprog to `--sectoranduicrerase`. This should ensure that each time we're flashing the validation firmware to a clean slate without needing to call a separate erase. As for ending up in normal mode, I'm guessing that you're calling flash_validation from this commit, which does not have the validation_manager module yet. If you run `nrfjprog --memrd 0x10001080` after calling flash_validation it should read "0xFFFFFFFE" which would indicate a success in these circumstances.

**Comment by kylzrnr** [comment]
I can't seem to recreate the error message you've mentioned here. I deleted the entire contents of my _build folder and called flash_validation without any errors.

**Comment by obasille** [comment]
My bad, since this was a new clone I needed to copy nrfutil.exe...
All good now!

**Review by obasille — APPROVED** [review]

**Pull request merged**